webpack.config.js

```diff
@@ -58,8 +59,55 @@ const config = withModuleFederationPlugin({
       requiredVersion: 'auto',
       includeSecondaries: true,
   },
+   '@onecx/angular-auth': {
```

Comment:
You should make sure that every package form package-lock.json starting with `@onecx/*` is shared. I think `@onecx/angular-accelerator` is missing.